hdl/axil_to_mcl.vh

```diff
@@ -38,9 +38,11 @@ package cl_mcl_pkg;
   parameter axil_s_fifo_base_addr_p = 64'h00000000_00001000;
   parameter axil_mon_base_addr_p = 64'h00000000_00002000;
 
-  parameter HOST_RCV_VACANCY_MC_REQ = 32'h100;
```

Comment:
(Meta comment)

@tommydcjung Is it "standard" to name package files with a _pkg suffix?

Reply:
yup

Reply:
Leonard, can you change the filename here?

Reply:
How about this ?
bsg_axil_to_mcl_pkg.vh